sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -37,7 +37,10 @@ export type AuthorizationRule = {
 // @public
 export interface CorrelationRuleFilter {
     applicationProperties?: {
-        [key: string]: string | number | boolean | Date;
+        [key: string]: string | number | boolean | Date | {
+            val: string;
```

Comment:
how would you feel about changing this from `val` to `value`? saving two characters doesn't seem worth the sacrifice in readability to me.

Reply:
Will do